### ISSUE TITLE
Prices copied back to purchase order are wrong

### ISSUE THREAD

**Issue by fietser28** [SergeoLacruz/inventree-supplier-panel#3]
If I add multiple items to my purchase order and press the 'Transfer PO' button:

The correct prices / amounts / etc. are displayed in the Mouser Actions panel:
<img width="1636" alt="image" src="https://github.com/SergeoLacruz/inventree-supplier-panel/assets/8796121/edab346b-84d1-479e-bd05-2f53a96cb62a">

But, the wrong prices are set in the purchase order:
<img width="1636" alt="image" src="https://github.com/SergeoLacruz/inventree-supplier-panel/assets/8796121/0dc7c568-e689-4488-ba8e-1110642f8b3a">
It seems one of the prices is set to all the line items in the purchase order.

Running inventree 0.13.0 and the latest version of this plugin.

BTW: The functionality of this plugin is really great, can't wait for TME and Digikey support.

**Comment by SergeoLacruz** [comment]
Thanks for the hint. This is because you do not have IPNs added to your parts. We use IPNs, so the problem never showed up. I changed the code line to use the SKU now which needs to be there. Give it a try.
I will never add TME because we do not have TME as a supplier. Digikey is still on the roadmap but their authentication system is quite complex. I tried to add Farnell but found that they do not have a shopping cart API so far.
